resources/functionMap.php

```diff
@@ -1988,7 +1988,7 @@
 'DOMNode::isDefaultNamespace' => ['bool', 'namespaceuri'=>'string'],
 'DOMNode::isSameNode' => ['bool', 'node'=>'DOMNode'],
 'DOMNode::isSupported' => ['bool', 'feature'=>'string', 'version'=>'string'],
-'DOMNode::lookupNamespaceURI' => ['string', 'prefix'=>'string'],
+'DOMNode::lookupNamespaceURI' => ['?string', 'prefix'=>'?string'],
```

Comment:
are you sure the prefix parameter is nullable?

per php.net docs it is not nullable
https://www.php.net/manual/en/domnode.lookupnamespaceuri.php

I don‘t know the details of the C php sources.. in case you are sure, please consider open a PR for the php.net docs

——

the return type seems fine

Reply:
Comment there says so: https://www.php.net/manual/en/domnode.lookupnamespaceuri.php#75556
Also in php-src master: https://github.com/php/php-src/blob/b3553159a7b15d1ddd485aba9f83485f08e2e800/ext/dom/php_dom.stub.php#L378-L379

Also, the referenced C sources don't change except for the `return;` (7.4.33) changing to `RETURN_THROWS();` (master).

Reply:
@paulbalandan please update the https://github.com/php/doc-en too